schedule/yast/maintenance/autoyast_create_hdd_maintenance_migration.yaml

```diff
@@ -0,0 +1,20 @@
+---
+name: autoyast_create_hdd_maintenance_migration
+schedule:
+  - autoyast/prepare_profile
+  - installation/isosize
+  - installation/bootloader
+  - autoyast/installation
+  - autoyast/console
+  - autoyast/login
+  - autoyast/wicked
+  - autoyast/repos
+  - autoyast/logs
+  - autoyast/autoyast_reboot
+  - installation/grub_test
+  - installation/first_boot
+  - console/system_prepare
+  - console/hostname
+  - update/patch_sle
```

Comment:
It seems works that you move patch_sle here for installation process, I just curious why patch_and_reboot not work.  
For me, it should work with patch_and_reboot in installation process, then add some split test modules instead of patch_sle should work.

Reply:
> It seems works that you move patch_sle here for installation process, I just curious why patch_and_reboot not work. For me, it should work with patch_and_reboot in installation process, then add some split test modules instead of patch_sle should work.

Now changed to use patch_and_reboot in 1st job and patch_sle in the 2nd job.
you can see [**patch_sle**](https://openqa.suse.de/tests/13725410#step/patch_sle/1) run 7 minutes while [**patch_and_reboot**](https://openqa.suse.de/tests/13725409#step/patch_and_reboot/1) used 2 minutes.

Reply:
@lemon-suse  please take a look again, thanks.

Reply:
Looks good to me.